program1.py

```diff
@@ -0,0 +1,77 @@
+"""
+this is first program to counte the number of vowels, words, sentences, and punctuation marks.
+"""
+
+from importlib.resources import contents
+from string import punctuation
+
+
+
+def count_vowels(test_Sentence):
+    number_of_vowels=0
+    """
+    this function is used to count vowels
+    Parameters:
+                    test_sentence (str): stores sentence set by user or data read from a file
+
+            Returns:
+                   number_of_vowels (int) : returns count of vowels in sentence
+    
+    """
+    for element in test_Sentence:
+        if element=="a" or element=="e" or element=="i" or element=="o" or element=="u":
```

Comment:
Can we somehow use the `in` operator of python to minimize these `if` conditions?